dashboard/lib/policies/lti.rb

```diff
@@ -90,11 +90,11 @@ module MessageType
     jwks_uri: CDO.studio_url('/oauth/jwks', CDO.default_scheme),
     token_endpoint_auth_method: "private_key_jwt",
     contacts: ["platform@code.org"],
-    scope: "https://purl.imsglobal.org/spec/lti-ags/scope/score https://purl.imsglobal.org/spec/lti-nrps/scope/contextmembership.readonly",
+    scope: ALL_SCOPES.join(' '),
     "https://purl.imsglobal.org/spec/lti-tool-configuration" => {
-      domain: "studio.code.org",
-      description: "Code.org",
-      target_link_uri: "https://studio.code.org/lti/v1/sync_course",
+      domain: CDO.dashboard_site_host,
+      description: "Code.org LTI Integration",
+      target_link_uri: CDO.studio_url('/lti/v1/sync_course', CDO.default_scheme),
```

Comment:
Does `lti_v1_sync_course_path` work for the path segment here? Could also add a new const DEFAULT_TARGET_LINK_URI and set it to `lti_v1_sync_course_path`

Reply:
`lti_v1_sync_course_path` doesn't seem to work here. I like the idea of adding a new const in this file and referencing it.